src/content/faq/tools/installing-dfu-util.md

```diff
@@ -0,0 +1,353 @@
+
+![Advanced Settings](/assets/images/installing-dfu-util-08advanced.png)
+
+Then click on **Environment Variables...**
+
+![Environment](/assets/images/installing-dfu-util-09environment.png)
+ 
+Click on **Path** in the bottom list (System variables) and click Edit.
+
+Finally, position the cursor at the end of the box and add to the end:
+
+```
+;C:\Program Files\dfu-util
+```
+There must be a semicolon separating the new item from the previous last item, and then the path to the directory we just created.
+
+![Edit variable](/assets/images/installing-dfu-util-19editvar.png)
+
+After editing the system path environment variable you'll need to restart the computer or log out and log back in.
+
+To test it, open a command prompt window and enter the command:
+
+```
+dfu-util -l
+```
+
+It should look something like this:
+
+![Edit variable](/assets/images/installing-dfu-util-20command.png)
+
+## Windows - Debugging drivers
+
+There are two separate device drivers in Windows, depending on whether the Particle device is in DFU or regular operating mode (or listening mode). 
+
+When in DFU mode, the Device Manager looks like this: (Note that it says **Photon DFU Mode** for all device types, including Electrons; this is normal.)
+
+![DFU driver](/assets/images/installing-dfu-util-22dfu.png)
+
+And when in normal operating mode or listening mode (blinking blue), the Device Manager looks like this:
+
+![COM driver](/assets/images/installing-dfu-util-23com.png)
+
+
+## Installation - Mac
+
+### Mac - Using homebrew
+
+If you have [homebrew](http://brew.sh) installed, it's as simple as running:
+
+```
+brew install dfu-util
+```
+
+If you have neither homebrew nor MacPorts installed, I recommend installing homebrew as the easiest way to install dfu-util. Just follow the instructions on the [homebrew](http://brew.sh) site to install it first.
+
+One thing about homebrew: it is designed to run without requiring superuser privileges (su). If you get a permission denied error, it's probably because your installation has incorrect permissions. You should stop and fix that first, because all sorts of strange things will continue to happen until you fix it. [This community forum post](https://community.particle.io/t/dfu-util-install-on-mac-os-x/18361) has more details on how to fix permission problems with homebrew.
+
+### Mac - Using MacPorts
+
+If you use [MacPorts](https://www.macports.org) you can install it using:
+
+```
+port install libusb dfu-util 
+```
+
+
+## Installation - Linux
+
+Most Linux installations have packages for dfu-util already. These are but a few examples:
+
+Ubuntu and Debian:
+
+```
+sudo apt-get install dfu-util
+```
+
+Fedora:
+
+```
+sudo yum install dfu-util
+```
+
+Arch:
+
+```
+sudo pacman -Sy dfu-util
+```
+
+You will also want to add a udev rule. This allows dfu-util to work without requiring sudo, which is convenient, and also necessary when using certain CLI and local build commands.
```

Comment:
This is done automatically now with the CLI if you run `particle update` or `particle flash --usb ...`, but would probably be good to have a manual way of adding it as well.